METRICS.md

```diff
@@ -105,6 +105,12 @@ memory_stats/stats/total_rss_huge | uint64 | The total number of bytes of anonym
 memory_stats/stats/total_swap | uint64 | The total number of bytes of swap usage <sup>(1)</sup>
 memory_stats/stats/total_unevictable | uint64 | The total number of bytes of memory that cannot be reclaimed <sup>(1)</sup>
 memory_stats/stats/total_writeback | uint64 | The total number of bytes of file/anon cache that are queued for syncing to disk <sup>(1)</sup>
+cpuset/cpu_exclusive | uint64 | contains a flag (0 or 1) that specifies whether cpusets other than this one and its parents and children can share the CPUs specified for this cpuset
+cpuset/memory_exclusive | uint64 | contains a flag (0 or 1) that specifies whether other cpusets can share the memory nodes specified for the cpuset
+cpuset/memory_migrate | uint64 | contains a flag (0 or 1) that specifies whether a page in memory should migrate to a new node if the values in cpuset.mems change
```

Comment:
Description is taken from official documentation. I'd prefer to leave it that way